### ISSUE TITLE
Перечисление пространств имён через запятую после using namespace

### ISSUE THREAD

**Issue by daniilrozanov** [cpp-ru/ideas#581]
Вместо того, чтобы тратить строчки с несколькими `using namespace X`, предлагается разрешить после директивы `using namespace` перечислять пространства имён через запятую.

Пример:

```cpp
// ...
void foo() {
  // using namespace ranges;
  // using namespace ranges::view;
  using namespace ranges, ranges::view;
  sort(arr, my_cmp); // ranges::sort
  for (X& x : arr | filter(&filters::even, &X::number)) // ranges::view::filter
    std::cout << x << std::endl;
}
```

**Comment by eoan-ermine** [comment]
`using` с C++17 поддерживает перечисление имен. Синтаксис `using namespace` в связи с этим выглядит асимметрично. Интересно, по какой причине вместе с `using` не завезли и `using namespace`.

**Comment by daniilrozanov** [comment]
> `using` с C++17 поддерживает перечисление имен. Синтаксис `using namespace` в связи с этим выглядит асимметрично. Интересно, по какой причине вместе с `using` не завезли и `using namespace`.

Видимо забыли

**Comment by AndreyG** [comment]
Нет, не забыли. Мотивацией для _using-declarator_-s была возможность использовать pack expansion (https://wg21.link/p0195), а вовсе не возможность сэкономить 5 букв на написание слова `using`. Для _using-directive_-s такой мотивации нет.
